jira/resources.py

```diff
@@ -482,6 +482,13 @@ def permalink(self):
     def __eq__(self, other):
         return self.id == other.id
 
+    def __getitem__(self, item):
+        # Grab the JIRA server
+        jira_server = self._session.jira_server
+
+        # Look for the appropriate fields list on the server
+        return getattr(self.fields, jira_server._fields[item])
```

Comment:
Hmm, very good point. Maybe there's some standard code that we can use to transform arbitrary text into a valid_python_identifier? (Because there's not just spaces but also punctuation and other cruft that could potentially be in custom field names.)